### ISSUE TITLE
file permission problem

### ISSUE THREAD

**Issue by rumpelsepp** [getpelican/pelican#1791]
I am the maintainer of the pelican AUR package: https://aur4.archlinux.org/packages/pelican
There is a problem with the file permissions in `/usr/lib/python3.4/site-packages/pelican/themes`. When using `setup.py install` these files have chmod 600, thus cannot be read by a regular user. A workaround is needed to change these permissions to 644: https://aur4.archlinux.org/cgit/aur.git/commit/?h=pelican&id=7d827041929070085bf40ae07834ba6e6d7adb35

**Comment by ingwinlu** [comment]
reproduceable, templates for tools are affected as well.

the only thing that touched packaging was bd5dfa0b3ec9560a1a453907f9f3767b9b89ea01.
@ionelmc any input on the matter?

**Comment by ingwinlu** [comment]
when i do a fresh sdist the permissions are correct for me, so the data_dir might be a dead end

**Comment by ionelmc** [comment]
The checkout is fine on my Ubuntu (775 for dirs and 664 for files).

Maybe there's something wrong with @rumpelsepp's umask or git settings?

**Comment by rumpelsepp** [comment]
> Maybe there's something wrong with @rumpelsepp's umask or git settings? 

It isn't.

```
$ umask
022
```

**Comment by ionelmc** [comment]
Perhaps there some issue with the setuptools installed on your system? What version do you have? Do other packages have this problem? (eg: try installing a package with data files)

**Comment by rumpelsepp** [comment]
I'm on arch linux, so there should be the most recent versions installed:

```
$ pacman -Q python-setuptools
python-setuptools 1:18.1-1
```
Maybe a setuptool issue? I ran into this the first time…

**Comment by ingwinlu** [comment]
the 3.6.2 tar file has the wrong permissions, trying to package master as sdist builds the tar file with the correct permissions on my machine. maybe something on @justinmayer 's end

**Comment by ionelmc** [comment]
Building the sdist outputs a tarball with correct perms on my machine. Maybe @justinmayer had wrong umask when he build it.

**Comment by justinmayer** [comment]
I released [3.6.3](https://github.com/getpelican/pelican/commit/ed83ad75a9aeaea97c6ab530b0679d1555c7691c) today, which should address this problem.
